src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = days * 40;
+
+  if (days >= 7) {
```

Comment:
Consider using a constant for the discount for rentals of 7 days or more (50) for better readability and easier maintenance. 🎁